test/e2e/chromesandbox_test.go

```diff
@@ -0,0 +1,146 @@
+// Copyright 2025 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/agent-sandbox/test/e2e/framework"
+)
+
+// TestRunChromeSandbox tests that we can run Chrome inside a Sandbox,
+// it also measures how long it takes for Chrome to start serving the CDP protocol.
+func TestRunChromeSandbox(t *testing.T) {
+	ctx := context.Background()
+	ctx, cancel := context.WithCancel(ctx)
+	defer cancel()
+
+	log := klog.FromContext(ctx)
+
+	h := framework.NewTestContext(t)
+
+	ns := fmt.Sprintf("chrome-sandbox-test-%d", time.Now().UnixNano())
+	h.CreateTempNamespace(ctx, ns)
```

Comment:
It looks like this function and `h.Apply` can both be implemented using `h.CreateWithCleanup`

Reply:
Switched CreateTempNamespace - thanks

Apply takes a yaml manifest, so that would be tricky